app/models/attribute.py

```diff
@@ -0,0 +1,28 @@
+from .. import db
+
+user_tag_associations_table = db.Table(
+    'user_tag_associations', db.Model.metadata,
+    db.Column('tag_id', db.Integer, db.ForeignKey('tags.id')),
+    db.Column('user_id', db.Integer, db.ForeignKey('users.id'))
+)
+
+resource_tag_associations_table = db.Table(
+    'resource_tag_associations', db.Model.metadata,
+    db.Column('resource_tag_associations', db.Integer,
```

Comment:
I think you probably accidentally changed this `db.Column('resource_tag_associations',...` by accident. Should still be `db.Column('tag_id',...`